base/linalg/blas.jl

```diff
@@ -65,6 +65,9 @@ const liblapack = Base.liblapack_name
 
 import ..LinAlg: BlasReal, BlasComplex, BlasFloat, BlasInt, DimensionMismatch, checksquare, axpy!
 
+const Complex64 = Complex{Float32}
+const Complex128 = Complex{Float64}
```

Comment:
I'm confused. Isn't the point to remove definitions like this?

Reply:
The BLAS module can define whatever convenience typealiases it wants though, right?

Reply:
Well sure, but it seems like for clarity it might be nice to use the more explicit type names.

Reply:
Yes, one reason I did that is that this file and in particular "lapack.jl" contain an awful lot of references to `Complex128/Complex64`, so I thougth it was fine to have local aliases (which I would certainly do for a personal project). Morever, I am really not familiar with the `LinAlg` module, so I prefer to receive suggestions from its authors as to what they prefer (alias or not). Also this module will eventually be split out from base, and so will these aliases.